### PR TITLE
Quick update of odo supported table

### PR DESCRIPTION
**Pull request by yhontyk**
- 4.2, 4.3, 4.4
- confirmed with the SME, basically, it is a direct copy-n-paste from the upstream;
 upstream PR: https://github.com/openshift/odo/pull/2687
 upstream issue: https://github.com/openshift/odo/issues/2683
 thanks @cdrage!

cc @bergerhoffer

### CODE REVIEW

File: cli_reference/openshift_developer_cli/understanding-odo.adoc [hunk 1]
```diff
@@ -40,18 +40,41 @@ In `{odo-title}`, services are provisioned from the OpenShift Service Catalog an
...
-|Node.js|Java
-|`centos/nodejs-8-centos7`|`redhat-openjdk-18/openjdk18-openshift`
-|`rhoar-nodejs/nodejs-8`|`openjdk/openjdk-11-rhel8`
-|`rhoar-nodejs/nodejs-10`|`openjdk/openjdk-11-rhel7`
-|`bucharestgold/centos7-s2i-nodejs`|
-|`rhscl/nodejs-8-rhel7`|
-|`rhscl/nodejs-10-rhel7`|
-|====
+|===
+|Language |Container Image |Package Manager
```
Comment (bergerhoffer) [inline cli_reference/openshift_developer_cli/understanding-odo.adoc]: Should be sentence case: "Container image" and "Package manager"

**Review by bergerhoffer — COMMENTED** [review]
Just one minor formatting comment, otherwise LGTM!

**Comment by bergerhoffer** [comment]
Just a reminder @boczkowska, it's best to ping the @peer-review-squad in the #team-docs channel for peer reviews. That way it gets spread to the entire OCP doc team and not bottlenecked only on me. CC @Preeticp too.

**Comment by Preeticp** [comment]
Ack @bergerhoffer we will keep that in mind, going forward.

**Comment by bergerhoffer** [comment]
Thanks for the update, merging!

**Pull request merged**